.github/workflows/on-pullrequest-reposcan.yml

```diff
@@ -16,4 +16,4 @@ jobs:
     name: Ed-Fi Repo Security Scan
     uses: ./.github/workflows/repository-scanner.yml
     with:
-      config-file-path: ./Ed-Fi-Actions/.github/workflows/bidi-excludes.json
+      config-file-path: ./.github/workflows/bidi-excludes.json
```

Comment:
This had led to a false-positive in the previous testing: this file path should be local to the current repository, without "knowing" where the repository will be checked out when running the action.